fabric-levelgenerators-v1/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-levelgenerators-v1"
```

Comment:
Not sure what the convention is here, is `levelgenerators` or `level-generators` better?

Reply:
`fabric-level-generators-v1` would fit the convention better.